# Phylogeny of Yellow Fever Virus, Uganda, 2016
## Introduction
In April 2016 , cases of yellow fever were identified in the southwestern district, Masaka, of Uganda . By June 2016, the Ministry 
of Health of Uganda had reported 68 suspected yellow fever cases, of which 3 probable and 7 confirmed cases were in the Masaka, 
Rukungiri, and Kalangala districts.
## Review
Presence of YFV RNA in human clinical samples confirmed using real-time reverse transcription PCR.
One serum sample was then selected for NGS because of its relative concentration of viral RNA (Determined by cycle threshold <30).The sample
was then sequenced using Ion torrent sequencing technology. 
This first round of  sequencing, resulted in none of the  sequence reads aligning to the YFV reference template indicative that the YFV
RNA in the sample was of low quality and/or quantity. The RNA sample was therefore subjected to a targeted RNase-H digestion 
to remove the contaminating carrier and ribosomal RNA. This step was performed so as to increase the sequencing coverage, probably
to increase chances of detecting infectious genetic material of YFV. A standard cDNA library was prepared and Ion torrent sequencing was performed. Fastq files generated were aligned again to YFV reference template using SeqMan NGen (DNASTAR).This second round of sequencing (after RNase-H treatment) resulted in 1.2% of all sequencing reads aligning to about 38% of the complete YFV genome.The longest and deepest coverage contigs were then taken through BLAST analyses. BlAST analyses determined that the Uganda 2016 strain is closely related to strains from South Sudan 2003 which confers to about 95% similarity (specifically in the envelope region).
This is in contrast to the similarity observed between Uganda 2016 and Angola 2016 which conferred to about 83% similarity in the same 
region. This result shows that the Uganda 2016 strain was not birthed by the Angola 2016 strain but rather the two outbreaks were 
independent of each other. The study goes further ahead to compare the NS genes of the Uganda 2016 with previous outbreaks. The finding
suggests that the NS genes (NS3 and NS5) of Uganda 2016 have the closest similarity to a Uganda 1948 strain. However it is worth noting that only specific regions of the YFV genome are being compared. Looking back at YFV outbreak history there seems to be 
insufficient full length YFV sequences from previous outbreaks. (Nina K stock 2013 et al). In my opinion this is one of the weak points of the study.
The study goes on to perform a phylogenetic analyses of the BLAST results and the result confirms the previously seen BLAST analyses. Uganda 2016 strain seems to cluster together in a clade consisting of other East African genotype strains whereas the angola 2016 strain clusters together with Angola 1971 YFV strain. This reiterates the fact that the Angola 2016 did not seed the Uganda 2016, rather the two outbreaks were independent of each other and most likely from different sources.
However tracing back the Uganda 2010 strain clustered with the East African/Central African genotype strains.(Using NGS to identify yellow fever virus paper). This is in contrast to the Uganda 2016 strain which clusters well with other East African genotypes.
This indicates that the Uganda 2016 strain and the Uganda 2010 strain belong to different genotypes and thus may not be from a common sylvatic origin. 
